sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventController.java

```diff
@@ -553,4 +559,30 @@ public void dearchiveAllSelectedItems(Collection<EventIndexDto> selectedRows, Ru
 				});
 		}
 	}
+
+	public VerticalLayout getEventViewTitleLayout(String uuid) {
+		EventDto event = findEvent(uuid);
+
+		VerticalLayout titleLayout = new VerticalLayout();
+		titleLayout.addStyleName(CssStyles.LAYOUT_MINIMAL);
+		titleLayout.setSpacing(false);
+
+		Label statusLabel = new Label(EventStatus.valueOf(event.getEventStatus().name()).toString());
+		statusLabel.addStyleNames(CssStyles.H3, CssStyles.VSPACE_NONE, CssStyles.VSPACE_TOP_NONE);
+		titleLayout.addComponents(statusLabel);
+
+		if (event.getStartDate() != null) {
```

Comment:
Events now have a start and end date, so this should rather reflect the date period (see DateHelper.buildPeriodString)